### ISSUE TITLE
[Bug Report] el-input-number.  Invalid prop

### ISSUE THREAD

**Issue by gezhigang1005** [element-plus/element-plus#3169]
### Element Plus version
1.1.0-beta.8

### OS/Browsers version
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.164 Safari/537.36

### Vue version
3.2.4

### Reproduction Link
https://github.com/element-plus/element-plus/blob/bfe23ce820f5da19730cadcc77b59f6846c6f70e/packages/components/input-number/src/index.vue#L103

### Steps to reproduce
1.1.0-beta.1 Previous version There's no problem

> Now
![image](https://user-images.githubusercontent.com/5440493/131629802-71650100-753a-45bc-8280-f1e9c457cd43.png)

### What is Expected?
```
modelValue: {
      required: true,
      validator: val => {
        return toRawType(val) === 'Number' || val === undefined
      },
},
```

### What is actually happening?
```
modelValue: {
      required: true,
      type: Number,
},
```

**Comment by jiaozhiye** [comment]
我也遇到了同样的困扰，希望官方能够尽快修复。

**Comment by Alanscut** [comment]
I have two questions:
  What scene needs to pass undefined
  What is the default value when passing undefined

**Comment by gezhigang1005** [comment]
@Alanscut 
Previous version There's no problem so It's all written that way：
```
const form = {}
```
```xml
<el-input-number v-model="form.num" />
```

**Comment by jiaozhiye** [comment]
场景1：初始状态输入框为空，没有初始值，此时的默认值是 undefined
场景2：即使初始状态有值，在用户手动清空后，此时的值是 undefined
在这两种场景下，都会出现警告

**Comment by jiaozhiye** [comment]
@Alanscut 
场景1：初始状态输入框为空，没有初始值，此时的默认值是 undefined
场景2：即使初始状态有值，在用户手动清空后，此时的值是 undefined
在这两种场景下，都会出现警告

**Comment by Alanscut** [comment]
@JeremyWuuuuu Setting a default value, such as 0, can solve this vue warn.However, considering whether it is necessary to add a new attribute, the user can pass a fixed value as the default value when undefined.

**Comment by msidolphin** [comment]
We have fixed this issue in #3190

**Comment by gezhigang1005** [comment]
@jiaozhiye   修复问题后还是有warn问题，看了一下代码就是把required给去掉了
